google-cloud-graalvm-samples/logging-sample/README.md

```diff
@@ -0,0 +1,32 @@
+# Cloud Logging Client Libraries with GraalVM
+
+This application uses the Google Cloud [Logging Client Libraries](https://github.com/googleapis/java-logging) and can be compiled with GraalVM Native Image.
+
+## Setup Instructions
+
+1. Follow the [GCP Project and GraalVM Setup Instructions](../README.md).
+
+2. Enable the [Logging APIs](https://console.cloud.google.com/flows/enableapi?apiid=logging.googleapis.com).
+
+## Run with GraalVM Compilation
+
+Navigate to this directory in a new terminal.
+
+1. Compile the application using the GraalVM Compiler. This step may take a few minutes.
+
+   ```
+   mvn package -P graal
+   ```
+
+2. Run the application:
```

Comment:
It looks like one may need to run `gcloud auth application-default login` to make the authorization work without a service account?